.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
```

Comment:
package-lock should be included. It is important for library installation.

Reply:
removed package-lock from gitignore.